source/base_interface.cc

```diff
@@ -207,6 +208,7 @@ void
 BaseInterface<dim,spacedim,LAC>::
 compute_system_operators(const std::vector<shared_ptr<LATrilinos::BlockMatrix> >,
                          LinearOperator<LATrilinos::VectorType> &,
+                         LinearOperator<LATrilinos::VectorType> &,
                          LinearOperator<LATrilinos::VectorType> &) const
```

Comment:
penso che dovremmo aggiungere un qualcosa che triggeri il calcolo del precondizionatore più fino. altrimenti in questo modo anche quando basta quello fino, quello più costoso viene calcolato ugualmente.

magari potremmo pensare una seconda funzione. dedicata solo al calcolo del finer preconditioner  che viene chiamata nel blocco catch

cosa ne pensate?